source/_data/glossary.yml

```
@@ -200,20 +200,15 @@
   link: /installation/#advanced-installation-methods
 - term: Home Assistant Core
   definition: >-
-    Home Assistant Core is a Python program. It can be run on various operating
-    systems and is the basis for Home Assistant. When people are talking about
-    Home Assistant Core they usually refer to a standalone installation method
-    that can be installed using a Virtual Environment or Docker. Home Assistant
-    Core does not use the Home Assistant Supervisor.
+    Home Assistant Core is the Python program at the heart of Home Assistant. It is part of all of the four installation types and can be run on various operating systems. It can be installed standalone (without Home Assistant Supervisor), using a Virtual Environment (typically referred to as Home Assistant Core installation method) or as a Container using Docker (typically referred to as Home Assistant Container installation method).
```

I guess this should be lower case? :thinking: 

```suggestion
    Home Assistant Core is the Python program at the heart of Home Assistant. It is part of all of the four installation types and can be run on various operating systems. It can be installed standalone (without Home Assistant Supervisor), using a Virtual Environment (typically referred to as Home Assistant Core installation method) or as a container using Docker (typically referred to as Home Assistant Container installation method).
```